scaloid-common/src/main/st/org/scaloid/common/preferences.scala

```diff
@@ -119,7 +121,7 @@ class Extra(val activity: SActivity) extends AnyVal with Dynamic {
         case v: Array[Double] => i.putExtra(name, v)
         case v: Array[String] => i.putExtra(name, v)
         case v: Array[CharSequence] => i.putExtra(name, v)
-        case v: Array[android.os.Parcelable] => i.putExtra(name, v)
```

Comment:
It's temporarily unqualified due to this Scalariform bug: https://github.com/daniel-trinh/scalariform/issues/44